content/posts/newsletter-009/index.md

```diff
@@ -51,6 +51,39 @@ some actual interesting discussions.
 If needed, a section can be split into subsections with a "------" delimiter.
 -->
 
+## [DUNGEONFOG] is Hiring
+
+![DUNGEONFOG editor](dgnfog.jpeg)
+
+[DUNGEONFOG] are developing editor tools for drawing and visualizing
+RPG tabletop maps.
+They're looking for a wgpu-rs developer for 2D graphics drawing.
+
+You can find all of the details on their [job offer page][dungeonfog-jobs].
+
+[DUNGEONFOG]: https://dungeonfog.com/
+[dungeonfog-jobs]: https://dungeonfog.com/about/job-offers/
+
+## [London Virtual Talks (GameDev Special)][ldn-video]
+
+[![TODO: describe](ldn-video.png)][ldn-video]
```

Comment:
Is this TODO still meant be be here? Don't know if you left it in by mistake or if you were going to come back to it before publishing :)

Reply:
thanks, fixed)